autogpt_platform/backend/backend/server/routers/v1.py

```diff
@@ -43,14 +52,10 @@ def execution_scheduler_client() -> ExecutionScheduler:
 v1_router = APIRouter(prefix="/api")
 
 
-v1_router.dependencies.append(Depends(auth_middleware))
-
-
 v1_router.include_router(
     backend.server.integrations.router.router,
     prefix="/integrations",
     tags=["integrations"],
-    dependencies=[Depends(auth_middleware)],
```

Comment:
don't love dropping this, but I get it.

Reply:
can't we use auth_middleware to validate the crypto too?

Reply:
Seems very out of scope to put that endpoint-specific and provider-specific piece of validation in the global `auth_middleware`. It's also not a given that all providers support such validation.

Reply:
oh is that provider specifc? i thought that was us generating a unique key that must be provided to prevent spoofing hooks by others

Reply:
Yeah but the way providers do the crypto and pass the hash isn't standardized. If in the future a pattern emerges we can always abstract it, but for now I don't see a common pattern (as there's only 1 `WebhooksManager` implemented lol)